courses/free-courses-es.md

```diff
@@ -291,6 +291,7 @@
 ### VS Code
 
 * [Visual Studio Code: Mejora tu velocidad para codificar](https://www.udemy.com/course/vscode-mejora-tu-velocidad-para-codificar/) - Fernando Herrera
+* [Aprende VS Code ahora! | curso completo de VSCode desde CERO](https://www.youtube.com/watch?v=Ei1y51K8jQk) - HolaMundo
```

Comment:
We prefer 'playlist' links for YouTube contributions. Also, please use '-' instead of '|' in your next contributions

Reply:
Ok, can i still please get the pull request still because I added more with playlists please

Reply:
also the | was in the title..

Reply:
replace it with `-` or` \|`
we encourage folks to submit atomic changes, using a fresh branch for each change. Easier to review, and fewer conflicts

Reply:
GIVE ME THE PULL REQUEST FOR HACKTOBERFEST PLEASE